scripts/manage-db.py

```diff
@@ -13,22 +14,74 @@
 
 from auslib.db import AUSDatabase
 
+def cleanup_releases(trans, nightly_age, dryrun=True):
+    # This and the subsequent queries use "%%%%%" because we end up going
+    # through two levels of Python string formatting. The first is here,
+    # and the second happens at a low level of SQLAlchemy when the transaction
+    # is being executed.
+    query = """
+LEFT OUTER JOIN releases_history USING (name)
```

Comment:
It's a touch hidden, but we actually do check releases_history.change_id in the last line of this query, which is used to make sure we don't delete any releases that are too recent.